.travis.yml

```diff
@@ -43,11 +47,20 @@ matrix:
     - os: osx
       env: BUILD_TYPE=Debug BUILD_EXTERNAL=0 MAKE_TARGETS="llvm-spirv" MAKE_TEST_TARGET="check-llvm-spirv"
       osx_image: xcode9.3
+
+    - env: BUILD_EXTERNAL=1 CHECK_FORMAT=ON
```

Comment:
I think it is better to use uniform syntax like `BUILD_EXTERNAL=1 CHECK_FORMAT=1` or `BUILD_EXTERNAL=ON CHECK_FORMAT=ON`

Reply:
Changed. I had kept the notation used by SPIRV-Tools